src/install/frontend.py

```diff
@@ -17,19 +17,19 @@
 def execute_commands_and_raise_on_return_code(commands, error=None):  # pylint: disable=invalid-name
     for command in commands:
         bad_return = error if error else 'execute {}'.format(command)
-        output, return_code = execute_shell_command_get_return_code(command)
+        output, return_code = execute_shell_command_get_return_code(command, timeout=10)
```

Comment:
It would probably better to make the timeout a parameter of the function `execute_commands_and_raise_on_return_code` so it can be adjusted as needed. E.g. the call to this function in `_build_highlight_js` should have a higher timeout as something gets downloaded  (maybe 30 seconds or 60) and for the other calls 10 seconds is probably enough.